utils/frame/try-runtime/cli/tests/follow_chain.rs

```diff
@@ -47,9 +44,8 @@ mod tests {
 			}
 
 			// Start a node and wait for it to begin finalizing blocks
-			let mut node = common::KillChildOnDrop(common::start_node());
-			let ws_url = common::extract_info_from_output(node.stderr.take().unwrap()).0.ws_url;
-			common::wait_n_finalized_blocks(1, &ws_url).await;
+			let (_child, ws_url) = common::start_node();
```

Comment:
For some reason, if I don't specify _child like this: 

```suggestion
			let (_, ws_url) = common::start_node();
```

Something happens to the node and it doesn't accept WS requests 🤷 

@brunopgalvao this should fix the issue you were having